### PR TITLE
Completing Party Name Format Fix

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#11742

*Description of changes:*
* Fixed Name Format

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/CompletingParty/CompletingParty.vue [hunk 1]
```diff
@@ -204,8 +204,8 @@ export default class CompletingParty extends Vue {
 
   private get getCompletingPartyName (): string {
     return (
-      this.currentCompletingParty.firstName +
-      this.currentCompletingParty.middleName +
+      this.currentCompletingParty.firstName + ' ' +
+      this.currentCompletingParty.middleName + ' ' +
```
Comment (severinbeauvais) [inline src/components/CompletingParty/CompletingParty.vue:208]: The spaces need to be conditional.

Comment (severinbeauvais) [reply src/components/CompletingParty/CompletingParty.vue:208]: Or you could `trim()` the result and then check if it's empty.

Comment (cameron-freshworks) [reply src/components/CompletingParty/CompletingParty.vue:208]: doh, of course

Comment (cameron-freshworks) [reply src/components/CompletingParty/CompletingParty.vue:208]: Updated to template with conditional middle

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**